src/components/CustomerQuota/CheckoutUnsuccessful/CheckoutUnsuccessfulCard.tsx

```diff
@@ -0,0 +1,135 @@
+ * - Redeemed token and returned token does not have the same id
+ * - Have not redeemed the token but attempted to return
+ */
+export const CheckoutUnsuccessfulCard: FunctionComponent<CheckoutUnsuccessfulCard> = ({
+  ids,
+  onCancel,
+}) => {
+  const { policies: allProducts } = useContext(CampaignConfigContext);
+  const { sessionToken, endpoint } = useContext(AuthContext);
+  const { pastTransactionsResult, loading, error } = usePastTransaction(
+    ids,
+    sessionToken,
+    endpoint
+  );
+  // Assumes results are already sorted (valid assumption for results from /transactions/history)
+  const sortedTransactions = pastTransactionsResult;
+
+  const { showErrorAlert } = useContext(AlertModalContext);
+  useEffect(() => {
+    if (error) {
+      showErrorAlert(error);
+    }
+  }, [error, showErrorAlert]);
+
+  const translationProps = useTranslate();
+  const transactionsByTimeMap = groupTransactionsByTime(
+    sortedTransactions,
+    allProducts || [],
+    translationProps
+  );
+  const transactionsByTimeList = sortTransactions(transactionsByTimeMap);
+
+  const { i18nt } = useTranslate();
+  return (
+    <View>
+      <CustomerCard ids={ids} headerBackgroundColor={color("red", 60)}>
+        <View
+          style={[
+            sharedStyles.resultWrapper,
+            sharedStyles.failureResultWrapper,
+          ]}
+        >
+          <FontAwesome
+            name="thumbs-down"
+            color={color("red", 60)}
+            style={sharedStyles.icon}
+          />
+          <AppText style={sharedStyles.statusTitleWrapper}>
+            <AppText
+              style={sharedStyles.statusTitle}
+              accessibilityLabel="checkout-unsuccessful-title"
+              testID="checkout-unsuccessful-title"
+              accessible={true}
+            >
+              {i18nt("checkoutUnsuccessfulScreen", "unsuccessful")}
+            </AppText>
+          </AppText>
+          <View>
+            <AppText style={{ marginBottom: size(1) }}>
+              {i18nt("checkoutUnsuccessfulScreen", "logAppeal")}
+            </AppText>
+          </View>
+          <View>
+            <AppText>
+              {`${i18nt(
+                "checkoutUnsuccessfulScreen",
+                "unsuccessfulRedeemAttempt"
+              )}:`}
+            </AppText>
+            <View style={styles.checkoutItemsList}>
+              {loading ? (
+                <ActivityIndicator
+                  style={{ alignSelf: "flex-start" }}
+                  size="large"
+                  color={color("grey", 40)}
+                />
+              ) : (
+                transactionsByTimeList.map(
+                  (transactionsByTime: TransactionsGroup, index: number) => (
+                    <TransactionsGroup
+                      key={index}
+                      maxTransactionsToDisplay={BIG_NUMBER}
+                      {...transactionsByTime}
+                    />
+                  )
+                )
```

Comment:
Remember to use the usual transaction list that displays a truncated list of transactions with the "See more..." link as discussed!